accounting/localizations/argentina.rst

```diff
@@ -54,11 +59,57 @@ many accounts as the companies that gave more complex fiscal requirements:
 Configure Master data
 ---------------------
 
+Electronic Invoice Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Environment
+***********
+
+The AFIP infrastructure is replicated in two separate environments, Testing and Production.
+
+Testing is provided so that the Companies can test their developments until they are ready to move 
+into the Production environment. As these two environments are completely isolated from each other, 
+the digital certificates of one instance are not valid in the other one. 
+
+Go to :menuselection:`Accounting --> Settings --> Argentinian Localization` to select the environment:
+
+.. image:: media/argentina_edi_01.png
+   :align: center
+
+
+AFIP Certificates
+*****************
+The electronic invoice and other afip services work with WebServices (WS) provided by the AFIP. 
+
+In order to enable communication with the AFIP, the first step is to request a Digital Certificate
+if you don’t have one already.   
+
+#. Generate certificate Sign Request (Odoo).
+   When this option is selected a file with extension .csr  (certificate signing request)
+   is generated to be used the AFIP portal to request the certificate. 
+
+  .. image:: media/argentina_edi_02.png
```

Comment:
Under-indented by 1 space since you're now using `#.` instead of `-` for the list elements. The image currently renders as a quote.

Reply:
Fix included in the last commit.